os/coreos-hardening-guide.md

```diff
@@ -56,3 +56,6 @@ SocketGroup=root
 EOM
 systemctl daemon-reload
 ```
+
+[etcd-ssh-guide]: customizing-sshd.md
+[etcd-sec-guide]: https://github.com/coreos/etcd/blob/master/Documentation/op-guide/security.md
```

Comment:
Link against a tag or ref. Branches change over time and the links can break on their own.